src/components/bulkConcepts/component/PreviewCard.jsx

```diff
@@ -31,7 +31,7 @@ const PreviewCard = ({
     minute: 'numeric',
   };
 
-  const mapping = mappings ? mappings.length : 'none';
```

Comment:
Is changing this line necessary ?

Reply:
Yeah. `MappingPreview` component  being called here`<MappingPreview title="Mappings" body={mapping} />` is supposed to show `mappings` but `const mapping = mappings ? mappings.length : 'none';` only sends the `number` of mappings available on a concept and `NOT` the mappings themselves. It was also causing an error because the react table in `MappingPreview` was not expecting a number.